src/examples/src/widgets/card/CardWithMediaContent.tsx

```diff
@@ -0,0 +1,27 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import Card from '@dojo/widgets/card';
+import * as cardCss from '../../../../theme/card.m.css';
+
+const factory = create();
+
+export default factory(function CardWithMediaContent() {
+	return (
+		<div styles={{ width: '400px' }}>
```

Comment:
We can have css files for the examples so I think any example styling should be added there.